extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -152,22 +171,22 @@ export function inflateAndSendRtc_(a4aElement, url, seenUrls, promiseArray,
     if (Object.keys(seenUrls).length == MAX_RTC_CALLOUTS) {
       return buildErrorResponse_(
           RTC_ERROR_ENUM.MAX_CALLOUTS_EXCEEDED,
-          opt_vendor || url, undefined, true);
+          callout, undefined, true);
```

Comment:
Let's always report errors... the only place where I don't see that we do is for malformed json and I can't see why we wouldn't.  Just remove opt_log and then you can remove all of these undefined portions for opt_rtcTime